client/state/login/actions.js

```diff
@@ -293,8 +294,7 @@ export const loginSocialUser = ( service, token, redirectTo ) => dispatch => {
 			dispatch( {
 				type: SOCIAL_LOGIN_REQUEST_FAILURE,
 				error,
-				service: service,
-				token: token,
+				authInfo: socialInfo
```

Comment:
Cause we are already in the `socialAccount` reducer, this property is simply the auth response from the social service that we are storing temporarily, makes more sense to call it `socialAccount.authInfo` in this case